### ISSUE TITLE
Empty label?

### ISSUE THREAD

**Issue by Jacobew** [google/youtube-8m#132]
Hi, I am not familiar with tensorflow and I extract the video-level features from tfrecord and convert them to torch.Tensor. 
However, I find a large proportion of videos (nearly 20% of them) have empty label. 
I wonder if it's because I made some mistakes in extraction or youtube-8m just organized in this way. 
Please correct me if I'm wrong! Here's a minimal script of extraction:
``` python
import tensorflow as tf
import numpy as np
from tqdm import tqdm
import os
import json
import torch
tf.compat.v1.enable_eager_execution()
split = ['train', 'validate', 'test']
video_files = []
root, store = './2/video/', './2/video_convert'
for d in os.listdir(root):
    video_files.append(os.path.join(root, d))

train_label, val_label, test_label = {}, {}, {}
nonzero, zero = 0, 0
total, video_cnt = 0, 0
for file in tqdm(video_files):
    try:
        for ss in split:
            if ss in file:
                _split = ss
        for example in tf.compat.v1.python_io.tf_record_iterator(file):
            tf_example = tf.train.Example.FromString(example)
            _id = tf_example.features.feature['id'].bytes_list.value[0].decode('utf-8')
            _label = list(tf_example.features.feature['labels'].int64_list.value)
            _rgb = tf_example.features.feature['mean_rgb'].float_list.value
            _audio = tf_example.features.feature['mean_audio'].float_list.value
            
            copy = _label[:]
            if _split == 'train':
                train_label[_id] = copy
            elif _split == 'validate':
                val_label[_id] = copy
            else:
                test_label[_id] = copy
            if len(copy) != 0:
                nonzero += 1
            else:
                zero += 1
            total += len(copy)
            video_cnt += 1
                
#             torch.save(torch.Tensor(_rgb), f'{store}/{_split}/{_id}_rgb.pt')
#             torch.save(torch.Tensor(_audio), f'{store}/{_split}/{_id}_audio.pt')
            
    except Exception as e:
        print('corrupt:', file, e)
print(nonzero, zero) # 5001275 1133323
print('average:', total/video_cnt) # average: 2.455081164242547
```

**Comment by limetus** [comment]
@Jacobew Hello Jacobew.

In my case, I fully download the dataset, but whole dataset size was not 1.5TB.
In this reason, when I extracted tfrecord file, all rgb is empty with train data.

Was it same with your case?

Thank you.

**Comment by Jacobew** [comment]
@limetus , Hi, everything went well in my case and I think this is because you had a corrupted copy of the dataset. You should re-download them and check whether the tfrecords are intact.

**Comment by limetus** [comment]
@Jacobew , Thank you for your kind comment. 

I downloaded **Video-level Training Data** with
```
curl data.yt8m.org/download.py | partition=2/frame/train mirror=us python
```

And then I checked the size of each train****.tfrecord file, it was about 250MB. (250MB * 3844tfrecord file = 960GB)

In your case, how big was the size of each train tfrecord file?

Thank you.

**Comment by Jacobew** [comment]
@limetus Yes each tfrecord is about 250MB. The total size of all splits (including validate and test) adds up to around 1.5TB.

**Comment by limetus** [comment]
@Jacobew , Oh I see.. I should include validate and test files. Thank you so much!
